deployment/common/changeset/link_transfer_timelock.go

```diff
@@ -0,0 +1,144 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type LinkTransfer struct {
+	To    common.Address
+	Value *big.Int
+}
+type LinkTransferTimelockRequest struct {
+	Transfers       map[uint64][]LinkTransfer
+	ValidUntil      uint32        // unix time until the proposal will be valid
+	MinDelay        time.Duration // delay for timelock worker to execute the transfers.
+	OverrideRoot    bool
+	StartingOpCount map[uint64]uint64
+	UseMCMS         bool
```

Comment:
I'd bundle all the MCMS info in a struct like this https://github.com/smartcontractkit/chainlink/blob/88a0338576dad34068f8ee278a5cf5ac0b8870ad/deployment/ccip/changeset/cs_update_rmn_config.go#L38, ValidUntil etc. don't make sense if its false

Reply:
Added here: https://github.com/smartcontractkit/chainlink/pull/15512/commits/ba6b57daeb9509733ba0e4adc4ff4229b7c1ac82